### PR TITLE
NewsWeek clone using boostrap 

### PR DESCRIPTION
**Pull request by akshay-narkar**
Using the bootstrap, the clone of NewsWeek was created. Different feature branches were created to work on multiple elements simultaneously.

### CODE REVIEW

File: stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,799 @@
...
+  margin-right: -15px;
+  margin-left: -15px;
+}
+
+/*
+nav* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+} */
```
Comment (euqueme) [inline stylesheet.css]: - [x] Delete the commented code

File: stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,799 @@
...
+
+.newsletter button {
+  flex: 1;
+  font-size: 16px;
+  color: #fff;
+  background: #f72210;
+  font-weight: 700;
+  border: none;
+
+  /* padding: auto 20px; */
```
Comment (euqueme) [inline stylesheet.css]: - [x] Delete the commented code

File: stylesheet.css [hunk 3]
```diff
@@ -0,0 +1,799 @@
...
+
+.width-small-screenak {
+  max-width: 65%;
+}
+
+/* sponsored */
+.sponsor-containak {
+  border: 1px solid #e6e6e6;
+
+  /* background-color: #f8f8f8; */
```
Comment (euqueme) [inline stylesheet.css]: - [x] Delete the commented code

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1344 @@
...
+                  <div class="top-story-container container-fluid left-story-containerak remove-widthak row">
+                    <div class="block-titleak">My Turn</div>
+
+                    <!-- first3 -->
+                    <div class="row remove-widthak col-md-12">
+                      <div class="continer-holder-eachak col-md-4 right-pad768adjak">
+                        <div class="story1ak culture">
+                          <!-- <div class="categoryak">
+              <a href="#">U.S.</a>
+            </div> -->
```
Comment (euqueme) [inline index.html]: - [x] Delete the commented code

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1344 @@
...
+                            <a href="#">'I'm a Lifelong Republican in Dixville Notch Who Just Voted for Biden' </a>
+                          </p>
+                        </div>
+                      </div>
+
+                      <div class="continer-holder-eachak col-md-4 rightleft-padadjak">
+                        <div class="story1ak culture">
+                          <!-- <div class="categoryak">
+              <a href="#">U.S.</a>
+            </div> -->
```
Comment (euqueme) [inline index.html]: - [x] Delete the comented code

File: index.html [hunk 6]
```diff
@@ -0,0 +1,1344 @@
...
+                            <a href="#">My Radio Audience Made Me Realize Why Hillary Clinton Lost Pennsylvania </a>
+                          </p>
+                        </div>
+                      </div>
+
+                      <div class="continer-holder-eachak col-md-4 left-pad768adjak">
+                        <div class="story1ak culture">
+                          <!-- <div class="categoryak">
+              <a href="#">U.S.</a>
+            </div> -->
```
Comment (euqueme) [inline index.html]: - [x] Delete the comented code

File: index.html [hunk 7]
```diff
@@ -0,0 +1,1344 @@
...
+                      <p class="eachstory-detailak each-story-textak">States across the country have seen issues with polling places, including whole counties experiencing power loss to stations.</p>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <!-- <div class="gray-border"></div> -->
```
Comment (euqueme) [inline index.html]: - [x] Delete the comented code

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request reviewed by Maria Eugenia Queme P. (TSE)

### Hello @akshay-narkar and @wandji20 :lollipop: :balloon: 

### Thank you for your submission :handshake: :handshake: You've done a great job most of your sections are looking just like the original :star_struck: so please feel very proud of your hard work :medal_sports: :medal_sports: let's address the following changes :construction: :construction_worker: :construction_worker: 

## Page Structure

### Navigation bar
#### Desktop screens

- [x] The page is displaying 2 navigation bars and the one belonging to this screen size should be centered let's fix it, also the NEWSWEEK logo should be white like the original

- index.html
image
- newsweek.com
image

### Subscribe and Newsletter
#### Desktop screens

- [x] These sections don't look like the original page let's fix them

- index.html
image
- newsweek.com
image

#### Tablet screens

- [x] These sections should be next to each other vertically in this screen size let's fix them

- index.html
image
-newsweek.com
image

#### Mobile screens

- [x] The SUBSCRIBE section is missing the overlapping image and all the contents should be centered

- [x] The NEWSLETTER section content should also be centered and the FREE SIGN UP> button should be aligned correctly

- index.html
image
- newsweek.com
image

## Project Specific
### Bootstrap use

- [x] This project is all about using bootstrap so make sure you use it for everything the less custom styling the better (it's better to have a big index.html file than big .css files)  the following are the directives I spot you used more in your stylesheet.css file and have a bootstrap option:

- width: 100%; (used 14 times) check [this](https://getbootstrap.com/docs/4.0/utilities/sizing/)
- color: black; (used 7 times) check [this](https://getbootstrap.com/docs/4.0/utilities/colors/#color)
- display: [value]; (used 25 times) check [this](https://getbootstrap.com/docs/4.0/utilities/display/)
- position: [value]; (used 14 times) check [this](https://getbootstrap.com/docs/4.0/utilities/position/)
- justify-content: [value]; (used 6 times) check [this](https://getbootstrap.com/docs/4.0/utilities/flex/#justify-content)
- text-transform: [value]; (used 16 times) check [this](https://getbootstrap.com/docs/4.0/utilities/text/#text-transform)
- font-weight: [value]; (used 27 times) check [this] (https://getbootstrap.com/docs/4.0/utilities/text/#font-weight-and-italics)
- text-align: [value]; (used 8 times) check [this](https://getbootstrap.com/docs/4.0/utilities/text/#text-alignment)
- background-color: white; (used 3 times) check [this](https://getbootstrap.com/docs/4.0/utilities/colors/#background-color)
- border-radius: [value]; (used 4 times) check [this](https://getbootstrap.com/docs/4.0/utilities/borders/#border-radius)
### CSS bad practices
> Make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

#### No repeating of the same rule

- [x] Not everything can be handled by Bootstrap but let's avoid repeating rules (keep your code DRY) and create classes that can be reused within your index.html file

- background-color: #f4f5f6; (used 3 times)
- margin-top: 20px; (used 2 times)
- padding-top: 20px; (used 3 times)
- font-family: "Roboto Condensed", sans-serif; (used 2 times)
- margin-bottom: 10px; (used 4 times)
- max-width: 1310px; (used 4 times)
- margin: 0 auto; (used 3 times)
- font-size: 12px; (used 2 times)
- font-size: 16px; (used 11 times)
- margin-bottom: 0; (used 3 times)
- padding-bottom: 10px; (used 2 times)
- font-size: 15px; (used 2 times)
- height: 32px; (used 2 times)
- color: #555; (used 2 times)
- font-size: 14px; (used 5 times)
- color: #666; (used 3 times)
- text-decoration: none; (used 5 times)
- line-height: 12px; (used 2 times)

### No commented code
> Comments: It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

- [x] I've put comments directly in your code where I found commented code let's delete it

### Readme file

- [x] Your Live demo is leading to a broken page let's fix it

### That's it, team! :raised_hands: 

### Please :pray: do the changes :technologist: :technologist: and submit back for a code review when you are done :memo: 

### Happy coding! :rocket: 

Any questions? ask me on slack **@ Maru** :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @akshay-narkar and @wandji20 :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**